Description: Return error, don't crash, if /dev/dri/card0 inaccessible

e.g. in chroots

Author: Rebecca Palmer <rebecca_palmer@zoho.com>

--- beignet-1.0.2.orig/src/x11/dricommon.c
+++ beignet-1.0.2/src/x11/dricommon.c
@@ -284,7 +284,6 @@ getDRI2State(Display* dpy, int screen, c
     goto err_out;
 
   fd = open(device_name, O_RDWR);
-  assert(fd >= 0);
 
   if (fd < 0)
     goto err_out;
